lib/split/alternative.rb

```diff
@@ -93,9 +121,9 @@ def control?
       experiment.control.name == self.name
     end
 
-    def conversion_rate(goal = nil)
-      return 0 if participant_count.zero?
-      (completed_count(goal).to_f)/participant_count.to_f
+    def conversion_rate(goal = nil, version_key = version)
+      return 0 if participant_count(version_key).zero?
+      (completed_count(goal, version_key).to_f)/participant_count(version_key).to_f
```

Comment:
It looks like there's an existing test for this function, but it's worth adding more to confirm that this functionality still works as expected when multiple versions are present.

Reply:
Also, I'm reading this as ```Conversion rate for version X of this alterative```.  Is the the correct interpretation?

Reply:
correct, It would do for me to clarify